src/measure_theory/borel_space.lean

```diff
@@ -1004,17 +1004,25 @@ end real
 
 variable [measurable_space α]
 
-lemma measurable.nnreal_of_real {f : α → ℝ} (hf : measurable f) :
-  measurable (λ x, nnreal.of_real (f x)) :=
+lemma measurable.real_to_nnreal {f : α → ℝ} (hf : measurable f) :
+  measurable (λ x, real.to_nnreal (f x)) :=
 nnreal.continuous_of_real.measurable.comp hf
 
+lemma ae_measurable.real_to_nnreal {f : α → ℝ} {μ : measure α} (hf : ae_measurable f μ) :
+  ae_measurable (λ x, real.to_nnreal (f x)) μ :=
+nnreal.continuous_of_real.measurable.comp_ae_measurable hf
+
 lemma nnreal.measurable_coe : measurable (coe : ℝ≥0 → ℝ) :=
 nnreal.continuous_coe.measurable
 
 lemma measurable.nnreal_coe {f : α → ℝ≥0} (hf : measurable f) :
   measurable (λ x, (f x : ℝ)) :=
 nnreal.measurable_coe.comp hf
 
+lemma ae_measurable.nnreal_coe {f : α → ℝ≥0} {μ : measure α} (hf : ae_measurable f μ) :
```

Comment:
This one will be renamed in a later PR to be less ambiguous?

Reply:
Yes, definitely!